tests/qauto/homework1.spec.js

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from "@playwright/test";
+
+
+test.describe('Registration form test', () => {
+
+    test.beforeEach(async ({page})=>{
+        await page.goto("https://qauto.forstudy.space/");
+        await page.locator(".btn-primary").click();
+        await expect(page.locator(".modal-content")).toBeVisible();
+    })
+
+  test('Positive', async ({ page }) => {
+    await page.locator("#signupName").fill("Justin");
+    await page.locator("#signupLastName").fill("Potter");
+    await page.locator("#signupEmail").fill("dns-jp@test.com");
+    await page.locator("#signupPassword").fill("qwertY@123");
+    await page.locator("#signupRepeatPassword").fill("qwertY@123");
+
+    const name = await page.locator("#signupName").evaluate(element => element.value);
+    const expectedName = "Justin";
+    expect(name).toBe(expectedName);
+
+    const lastName = await page.locator("#signupLastName").evaluate(element => element.value);
+    const expectedLastName = "Potter";
+    expect(lastName).toBe(expectedLastName);
+
+    const email = await page.locator("#signupEmail").evaluate(element => element.value);
+    const expectedEmail = "dns-jp@test.com";
+    expect(email).toBe(expectedEmail);
+
+    const passwordValue = await page.locator("#signupPassword").evaluate(element => element.value);
+    const expectedValue = "qwertY@123";
+    expect(passwordValue).toBe(expectedValue);
+
+    const passworRepeatdValue = await page.locator("#signupPassword").evaluate(element => element.value);
```

Comment:
use https://playwright.dev/docs/api/class-locator#locator-inner-text
or even better
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

Reply:
still not resolved